.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I'm really not sure about that. I know I'm super picky with external/local imports ahah but you end up with this kind of stuff https://github.com/ueno-llc/ueno-next-starter/blob/0566a2e2aa81a7159052e45a1d400a40e11a641a/src/components/app-layout/AppLayout.tsx that looks odd to me

Reply:
A reason why this is better than our current setup is
a) it's automatic, which I prefer over having to manually sort and group imports
b) imports are just imports, we just import once and don't have to deal with it again
c) our current import grouping sucks because of the root resolver and collisions with real packages. The plugin was originally created to separate absolute and relative imports

Also regarding the root resolver:
```tsx
import { validate } from 'graphql/validation';
import { metaFragment } from 'graphql/fragments/meta';
```

This can be ambiguous as well.